oracle/src/oracleimporter.cpp

```diff
@@ -84,10 +84,14 @@ CardInfoPtr OracleImporter::addCard(const QString &setName,
 
         // detect mana generator artifacts
         bool mArtifact = false;
-        if (cardType.endsWith("Artifact"))
-            for (int i = 0; i < cardTextRows.size(); ++i)
-                if (cardTextRows[i].contains("{T}") && cardTextRows[i].contains("to your mana pool"))
+        if (cardType.endsWith("Artifact")) {
+            for (int i = 0; i < cardTextRows.size(); ++i) {
+                cardTextRows[i].remove(QRegularExpression("\\\".*?\\\""));
```

Comment:
Isn't `.*?` redundant? Shouldn't it just be `.*`?

Reply:
`.*?` means match any character any number of times, whereas `.*` means to match any character once... Not sure which is better here